src/commands/createNewProject/ProjectCreateStep/ScriptProjectCreateStep.ts

```diff
@@ -79,7 +79,15 @@ local.settings.json`));
             return {};
         } else {
             const hostJson: IHostJsonV2 = {
-                version: '2.0'
+                version: '2.0',
```

Comment:
This feels like a pretty obscure setting to me. I assume there's a reason this needs to be defaulted in the tooling instead of in the platform? If so - can you add a link to the original discussion in a comment (a TypeScript comment in the file itself, not a PR comment on GitHub)

Reply:
Found the [discussion](https://github.com/Azure/azure-functions-templates/issues/906). Reason was it's a breaking change if we default. Will add a comment.